apps/bestofjs-nextjs/src/app/api/og/route.tsx

```diff
@@ -35,7 +35,7 @@ export async function GET() {
   );
 }
 
-function ProjectRow({
+export function ProjectRow({
   project,
   rank,
 }: {
```

Comment:
Thank you for the PR!
You should move this function to a shared file as I think you cannot use named `export` for special files such as `page.tsx` or `route.tsx`.

I think it's the reason why the deploy failed on Vercel.

```
[16:27:04.943] Failed to compile.
[16:27:04.943] 
[16:27:04.943] src/app/api/og/route.tsx
[16:27:04.944] Type error: Route "src/app/api/og/route.tsx" does not match the required types of a Next.js Route.
[16:27:04.944]   "ProjectRow" is not a valid Route export field.
[16:27:04.944] 
[16:27:05.052]  ELIFECYCLE  Command failed with exit code 1.
[16:27:05.061] Error: Command "pnpm build" exited with 1
```